CHANGELOG.rst

```diff
@@ -7,6 +7,14 @@
 Changelog
 =========
 
+2.5.2 - 2023-05-25
```

Comment:
Do we need a release just for this bug fix? It doesn't seem that urgent.

Reply:
Agreed. Should I merge it with #641?

Reply:
No need – just change the date to "unreleased" (cf. #636). :)

Reply:
Ah okay, thanks!